WPMedia/fusion-news-theme-blocks
# [Themes-1688] Fixed inconsistent image width

## Description

#### Jira Ticket: [THEMES-1688](https://arcpublishing.atlassian.net/browse/THEMES-1688)

- Headlines in small and medium promos on mobile take up 66% container width
- Width of images in small and medium promos on mobile are the same in top table list

## Test Steps

1. Checkout this branch `git checkout THEMES-1688`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/top-table-list-block`
3. Add medium and small promos and then inspect to check the width


[THEMES-1688]: https://arcpublishing.atlassian.net/browse/THEMES-1688?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
